behavior.d.ts

```diff
@@ -207,7 +206,7 @@ declare function getCompReg(component_index: Value | CompNum, group_index?: Valu
 /**
  * Sets the numerical/coordinate part of a value
  */
-declare function setNumber(value: Value, num_coord: Value | CoordNum): Value;
```

Comment:
Why do you need this change?

Reply:
Because I want to be able to call
foo = setNumber(bar, 3);

Reply:
Isn't that already possible? 
```ts
type CoordNum = Coord | number;
```

Reply:
You are right, I missed something. Removed.